apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -908,7 +908,7 @@ export class RushConfiguration {
     this._yarnCacheFolder = path.resolve(path.join(this._commonTempFolder, 'yarn-cache'));
 
     this._changesFolder = path.join(this._commonFolder, RushConstants.changeFilesFolderName);
-    this._rushUserFolder = path.join(Utilities.getHomeDirectory(), '.rush');
```

Comment:
Is the **rush-recycler** under this folder?  It shouldn't be, since we want to ensure it will get cleaned up by any invocation of Rush.

Reply:
It currently is. I'll fix that.

---
In reply to: [231272392](https://github.com/Microsoft/web-build-tools/pull/934#discussion_r231272392) [](ancestors = 231272392)